Cargo.toml

```diff
@@ -15,7 +15,8 @@ dotenv = "0.15"
 env_logger = "0.9"
 fang = "0.6"
 feed-rs = "1"
-frankenstein = { version = "0.19", default-features = false, features = ["telegram-trait"] }
+frankenstein = { version = "0.19", default-features = false, features = ["async-http-client" , "async-telegram-trait"] }
```

Comment:
let's use both async and blocking clients.

async client will be used in the handler, blocking client will be used in background jobs